pkg/secrethub/client.go

```diff
@@ -52,6 +53,13 @@ type ClientInterface interface {
 
 var (
 	errClient = errio.Namespace("client")
+
+	whitelistAppInfoName = regexp.MustCompile("^[a-zA-Z0-9_-]*$")
```

Comment:
If we start validating this: do we want a maximum length?

Reply:
Set it to 64 chars now.